src/host/host.cpp

```diff
@@ -45,17 +45,21 @@ class RPCHandler : virtual public RPCIf {
         assert(argc >= 2);
 
         char *oe_enclave_path = argv[1];
+        uint32_t simulate_flag = OE_ENCLAVE_FLAG_DEBUG;
+        
         if (check_simulate(argc, argv)) {
             #ifdef DEBUG
             spdlog::debug("Running in simulation mode");
             #endif
 
-            oe_result_t result =
-                oe_create_ortoa_enclave(oe_enclave_path, OE_ENCLAVE_TYPE_SGX,
-                                        OE_ENCLAVE_FLAG_SIMULATE, NULL, 0, &enclave);
-            if (result != OE_OK) {
-                throw OECreationFailed(oe_enclave_path);
-            }
+            simulate_flag = OE_ENCLAVE_FLAG_SIMULATE;
```

Comment:
Look at line 46 here https://github.com/openenclave/openenclave/blob/e9a0423e3a0b242bccbe0b5b576e88b640f88f85/samples/pluggable_allocator/host/host.cpp

Reply:
Updated to use bitwise or